src/components/CollectionDetailPage/CollectionContextMenu/CollectionContextMenu.tsx

```diff
@@ -1,11 +1,13 @@
 import * as React from 'react'
-import CopyToClipboard from 'react-copy-to-clipboard'
 import { Dropdown, Button, Icon, Popup } from 'decentraland-ui'
 import { t } from 'decentraland-dapps/dist/modules/translation/utils'
+
 import { RoleType } from 'modules/collection/types'
 import { isOwner as isCollectionOwner, isLocked } from 'modules/collection/utils'
 import ConfirmDelete from 'components/ConfirmDelete'
 import { Props } from './CollectionContextMenu.types'
+import CopyToClipboard from 'components/CopyToClipboard/CopyToClipboard'
```

Comment:
Would you mind moving this import up so the relative imports are below?
```suggestion
import ConfirmDelete from 'components/ConfirmDelete'
import CopyToClipboard from 'components/CopyToClipboard/CopyToClipboard'
import { Props } from './CollectionContextMenu.types'
```